build/source/engine/run_oneGRU.f90

```diff
@@ -79,141 +68,142 @@ module run_oneGRU_module
  ! simulation for a single GRU
  subroutine run_oneGRU(&
                        ! model control
-                       iGRU,               & ! intent(in):    GRU index
```

Comment:
Do you want to pass the `iGRU` still into `run_oneGRU` simply for logging and error reporting purposes. In RASM we have had the problem in earlier versions that we would have an error in a low level routine and we wouldn't know where the error happened. If we get a complete stack trace when we report the error then that is not a problem. (note that it is a good sign that you do not need this info AT ALL to run the routine)

Reply:
Or is this part of `gruInfo`?

Reply:
We can get a traceback without including iGRU

Reply:
Perfect!

Reply:
The routine will exit gracefully, and then IGRU will be available.